src/api.js

```diff
@@ -2399,7 +2396,7 @@ api.declare({
         workerId: worker.workerId,
         firstClaim: worker.firstClaim.toJSON(),
         latestTask: worker.recentTasks.pop(),
-        disabled: worker.disabled,
+        quarantineUntil: worker.quarantineUntil.toJSON(),
```

Comment:
we should probably omit this when the date is in the past, so that users can simply check for existence when wondering if a worker is currently quarantined.

Reply:
Also, could this use `worker.json()`?

Reply:
Did this get missed?